CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 ## IN PROGRESS
 * Upgraded datastax driver to 3.1.2
+* Added new configuration option DELAYED_METRICS_REROLL_GRANULARITY
```

Comment:
Once everything is merged to master, I plan on writing a wiki page just for delayed metrics explaining various configs and how to tweak them.